Source/Particles/Collision/BinaryCollision/DSMC/DSMCFunc.cpp

```diff
@@ -52,29 +52,54 @@ DSMCFunc::DSMCFunc (
         WARPX_ALWAYS_ASSERT_WITH_MESSAGE(process.type() != ScatteringProcessType::INVALID,
                                         "Cannot add an unknown scattering process type");
 
-        m_scattering_processes.push_back(std::move(process));
-    }
+        // if the scattering process is ionization get the secondary species
+        // only one ionization process is supported, the vector
+        // m_ionization_processes is only used to make it simple to calculate
+        // the maximum collision frequency with the same function used for
+        // particle conserving processes
```

Comment:
Hm, I copied this from `BackgroundMCC.ccp` but it makes no sense here since we are *not* calculating the maximum collision frequency...